src/lib/components/commons/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  CURRENT_COLOR_CLASS_NAME,
+  POINTER_CURSOR_CLASS_NAME,
+} from '../../icons/SvgIcon';
+import {
+  StyledSearchBar,
+  StyledSearchInput,
+  StyledSearchIcon,
+} from './SearchBar.styled';
+import { ISearchBarProps } from './SearchBar.types';
+
+export const SearchBar = (props: ISearchBarProps) => {
+  const { setSearchValue } = props;
+  return (
+    <>
+      <StyledSearchBar>
+        <StyledSearchInput
+          name='search'
+          placeholder='חפש...'
+          onChange={(e) => {
+            setSearchValue(e.target.value);
+          }}
+        />
+        <StyledSearchIcon
+          {...props}
+          viewBox='0 0 32 32'
+          className={POINTER_CURSOR_CLASS_NAME}
+          size='S'
+        >
+          <path
```

Comment:
what is that? nested icon inside component? i don't like this

Reply:
i changet it.
it was something i tried and forgot about it..